src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length - 1; i++) {
+      if (compareFunction(this[i], this[i + 1]) > 0) {
```

Comment:
The condition `compareFunction(this[i], this[i + 1]) > 0` is correct for a bubble sort, but ensure that the compare function returns a number as expected by this condition.